app/src/main/java/com/fastaccess/ui/modules/settings/SettingsFragment.java

```diff
@@ -74,10 +81,20 @@
             restartActivity();
             return true;
         } else if (preference.getKey().equalsIgnoreCase("appTheme")) {
+            if(appTheme.equalsIgnoreCase(newValue.toString()))
```

Comment:
this will result to `NPE` to get around it we should check if the  newValue equals the oldValue which the old value might be null.

```if (newValue.toString().equalsIgnoreCase(appTheme)) return true;```